src/Util/Util.php

```diff
@@ -230,4 +230,13 @@ public static function buildDOMDocumentFromHtml( string $html ): DOMDocument {
 		$document->encoding = 'UTF-8';
 		return $document;
 	}
+
+	/**
+	 * Remove reserved characters from a cache key.
+	 * @param string $key
+	 * @return string
+	 */
+	public static function sanitizeCacheKey( string $key ): string {
+		return preg_replace( '/[{}()\/\@\:"]/', '', $key );
```

Comment:
Should probably replace with a hyphen or something, to avoid a completely empty key. (As I did somewhere recently, much annoyingly.)

Reply:
Done